scan/src/rpc/ScriptHook.re

```diff
@@ -0,0 +1,41 @@
+module Script = {
+  type field = {
```

Comment:
Let's name this `field_t`. It's a bit confusing that we declare `field` name here and also `field`  from `JsonUtils.Decode` below.